src/views/Teams/AddTeam.test.jsx

```diff
@@ -0,0 +1,74 @@
+import { render, screen } from '@testing-library/react';
+import { setupServer } from 'msw/node';
+import { rest } from 'msw';
+import { Route, Router } from 'react-router-dom';
+import userEvent from '@testing-library/user-event';
+import { createMemoryHistory } from 'history';
+import AddTeam from './AddTeam';
+import TeamDetails from './TeamDetails.jsx';
+
+const mockTeam = {
+  id: 6,
+  created_at: '2021-12-08T20:26:24.408898+00:00',
+  name: 'redirect me!',
+  city: 'okay',
+  state: 'go',
+  players: []
+};
+
+const server = setupServer(
+  rest.get(
+    'https://bmvgpxsbwlwuetqtoczm.supabase.co/rest/v1/teams',
+    (req, res, ctx) => {
+      return res(ctx.json(mockTeam));
+    }
+  ),
+  rest.post(
+    'https://bmvgpxsbwlwuetqtoczm.supabase.co/rest/v1/teams',
+    (req, res, ctx) => {
+      return res(ctx.json([mockTeam]));
+    }
+  )
+);
+
+beforeAll(() => {
+  server.listen();
+});
+
+afterAll(() => {
+  server.close();
+});
+
+it('should add a team and redirect to the team detail page', async () => {
+  const history = createMemoryHistory();
+  history.push('/teams/add');
+
+  render(
+    <Router history={history}>
+      <Route path='/teams/add'>
+        <AddTeam />
+      </Route>
+      <Route path='/teams/:id' component={TeamDetails} />
+    </Router>
+  );
+
+  screen.getByText(/Add Team/i);
+
+  const nameField = screen.getByLabelText(/name/i);
+  const cityField = screen.getByLabelText(/city/i);
+  const stateField = screen.getByLabelText(/state/i);
+  const submitBtn = screen.getByRole('button', { name: /Add team/i });
+
```

Comment:
yes, for the context of an `AddTeam` test, the pretend typing and button clicking don't do anything other than prove that things _can_ be typed and clicked. but! if you are searching for more meaning (as I suspect you are), you can use them in a test that confirms that after the button is clicked, some _new, edited_ information is displayed on the page!
I left some detailed comments on Zack's PR, but here's the gist:
```js
// have your userEvent stuff here
mockTeam.name = 'some edited name';
mockTeam.city = 'some edited city'
mockTeam.state = 'some edited state'
// now click that button! your rendered page should re-render and the mock server should provide the new, edited team information
// finally, you can `findByText(/some edited team/i)` to prove that the new, edited info is being displayed 
```